server.js

```diff
@@ -13,12 +13,22 @@ global.Discord = require('discord.js');
 const fs = require('fs');
 const express = require('express');
 const app = express();
+require('heroku-self-ping').default(config.herokuUrl);
 
 const port = process.env.PORT || 3000;
+
+// HTTP request for pinging
+//
+// Used to circumvent Heroku idling policy
+app.get('/ping', function(req, res){
```

Comment:
Free Heroku dynos have an uptime restriction (see [here](https://devcenter.heroku.com/articles/free-dyno-hours)), so if you keep pinging it to keep it up, you're gonna run out of those hours quickly and take down the other apps we have running on Heroku as well. Either we pay for the Heroku dyno that runs the bot or we look elsewhere for hosting.

Reply:
AWS or GCP, I suppose, but if not, I suppose we'll just hit the "/ping" endpoint at 4:29 AM using a cronjob in order to maintain the channel wipe functionality.

Reply:
I've removed the self-ping functionality for now, check c661598.